dotnet/samples/GettingStartedWithAgents/Step08_Declarative.cs

```diff
@@ -30,6 +31,35 @@ public async Task ChatCompletionAgentWithKernelAsync()
         await InvokeAgentAsync(agent!, "Cats and Dogs");
     }
 
+    [Fact]
+    public async Task ChatCompletionAgentWithFunctionsAsync()
```

Comment:
`Async` prefix is not needed anymore for our demos.

```suggestion
    public async Task ChatCompletionAgentWithFunctions()
```

Reply:
It's still needed in this getting started project. All of the other samples use it and removing the async suffix results in: Naming rule violation: Missing suffix: 'Async'